### DISCUSSION TITLE
Do not backup if system is running on battery

### DISCUSSION THREAD

**Discussion by jose1711** [borgbase/vorta#990] [Ideas]
Maybe add an option to skip/postpone backup if system is running on battery (and battery level is under certain threshold)? Backup means IO and you may simply want to work longer than have a most-recent backup.

**Comment by m3nu** [comment]
Do you know of a OS-independent way to get the current battery status? Else we would need different ways for each OS, similar to network or keychain.

**Reply by jose1711** [reply]
`psutil` looks to be quite neat (https://psutil.readthedocs.io/en/latest/#psutil.sensors_battery) and seems to work fine on my laptop. Maybe this could be a new optional dependency. If the module is not present - the checkbox to toggle the option would be disabled.

**Reply by m3nu** [reply]
We already use psutil to find existing Borg mounts. So this might be easy to add.

**Reply by bk138** [reply]
Hi guys, is there a tracking issue for this? (I used the search but was led here...)

**Reply by real-yfprojects** [reply]
Now there is: #1529
